local/sites/test-site/inventory.yml

```diff
@@ -35,12 +35,24 @@ name: etcd3
 interfaces:
   - type: physical
     subnet: vagrant1
-    bmc:
-      type: ssh
-      username: core
-      keypath: /vagrant/provision_files/id_rsa
     mac: 00:00:00:00:00:23
     ipv4: 10.10.10.23
+
+  - type: bmc
```

Comment:
lets remove this, we want all the vagrant VMs to be ssh reboots since ipmi will not work in virtualbox. 

If we want to just provide this as an example, lets put it on `master` for [vaquero-examples](https://github.com/CiscoCloud/vaquero-examples)